src/api/tiering_policy_api.js

```diff
@@ -217,5 +218,23 @@ module.exports = {
                 'OPTIMAL'
             ]
         },
+        tiering_policy_mode: {
```

Comment:
Is it the same as bucket_api::bucket_mode? 
Should we have a common type for both?